tests/Doc/HomeJsonExample.cs

```diff
@@ -0,0 +1,176 @@
+// EXAMPLE: cs_home_json
+
+// STEP_START import
+using NRedisStack.RedisStackCommands;
+using NRedisStack.Search;
+using NRedisStack.Search.Aggregation;
+using NRedisStack.Search.Literals.Enums;
+using NRedisStack.Tests;
```

Comment:
would move this into 'HIDE' region.
at first sight i was thinking this should go into 'REMOVE' region, but then i realized
we left some test project related stuff in 'HIDE' regions overall the docs, is it on purpose ?

Reply:
> would move this into 'HIDE' region. at first sight i was thinking this should go into 'REMOVE' region, but then i realized we left some test project related stuff in 'HIDE' regions overall the docs, is it on purpose ?

@atakavci Yes, I've done this on purpose for this example. It's actually for this [introductory example](https://redis.io/docs/latest/develop/connect/clients/dotnet/#example-indexing-and-querying-json-documents) on the NRedisStack "home" page. The existing example is written to mention everything right from the start (including imports, etc). You are right, though, that in most other examples, we try to focus on just the relevant commands, not the boilerplate.